### DISCUSSION TITLE
The campaign rarely starts in the 2nd biome (Starter Outpost)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14138] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

The game rarely fails to generate a basic "outpost" at the start of the first biome, and will start your campaign ahead on the map on the first outpost it finds

Example using Dynamic Europa, since we have many possible outpost types there's a non-zero chance this happens:

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/01b38eaa-b1b8-460f-b832-c5d70eb7d324)

**Please force the tutorial outpost in both SP and MP. This also allows customizeable starter outposts**

----------

### Reproduction steps

1. Download Dynamic Europa
https://steamcommunity.com/sharedfiles/filedetails/?id=2532991202
2. Roll a new campaign a dozen of times (with tutorial disabled) until this happens

or use this save 
[Save 54.zip](https://github.com/user-attachments/files/15846339/Save.54.zip)

### Bug prevalence

Happens every now and then

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

**Comment by SomeRandomNoobKekeke** [comment]
Just use `z6pKxwBx` seed in Dynamic europa
As i [said](https://github.com/FakeFishGames/Barotrauma/discussions/13791#discussioncomment-9173272) i've never seen anything like that in vanilla and i'm still too lazy to codedive or debug this

**Comment by Jasontti** [comment]
Tested the fix.
Closing discussion as completed.
